ValveResourceFormat/ThirdParty/DDSImage.cs

```diff
@@ -258,6 +258,13 @@ private static void DecompressBlockDXT5(int x, int y, int width, byte[] blockSto
                             finalG = ClampColor(finalAlpha + cg);
                             finalB = ClampColor(finalAlpha - co - cg);
                         }
+                        if (dxt5nm)
+                        {
+                            finalR = ClampColor(255 - finalAlpha);
```

Comment:
I don't think these need to be clamped.

Reply:
It's because of this:  
>If you treat it as YCoCg, and just replace alpha with 255 already gives a better output, but it's very bluish.
>It's the same with the formulas for CoCg_Y, so I went along with something else __since it more closely resembles tangent space normalmaps__.

Reply:
That's unrelated. What I mean is, the max value per channel is going to be `255`, so if you substract it from 255, it can't be out of 0-255 range.

Reply:
Fixed.